test/ui-test/src/screens/SettingsScreen.py

```diff
@@ -417,9 +417,9 @@ def check_backup_seed_phrase_workflow(self):
         started_at = time.monotonic()
         while wait_for_is_visible(BackupSeedPhrasePopup.REVEAL_SEED_PHRASE_BUTTON.value, verify=False):
             try:
-                hover(BackupSeedPhrasePopup.REVEAL_SEED_PHRASE_BUTTON.value)
+                # hover(BackupSeedPhrasePopup.REVEAL_SEED_PHRASE_BUTTON.value)
```

Comment:
Could you please delete it? I tested it on my local machine it also works well without a hover

Reply:
yes, sure, done!